<launch>
  <!-- Load RSSI Params from params/rssi.yaml -->
  <rosparam command="load" file="$(find omnibot)params/rssi.yaml" />
  <!-- Parameter Model -->
  <!-- 'rssi': {
      'accessPoints': {
          'ap1': {
              'signalAttenuation': 3, 
              'location': {
                  'y': 7, 
                  'x': 1
              }, 
              'reference': {
                  'distance': 5, 
                  'signal': 60
              }, 
              'name': 
              'dd-wrt'
          }, 
          'ap2': {
              'name': 'OnePlus3', 
              'location': {
                  'y': 11, 
                  'x': 3
              },
              'reference': {
                  'distance': 4, 
                  'signal': 40
              }, 
              'signalAttenuation': 3
          },
      }
      'networkInterface': 'wlp1s0'
  } -->
  <!--  Start rssiPublisher Node-->
  <!-- <node name="rssiPublisher_node" pkg="omnibot" type="rssiPublisher_node.py" /> -->
</launch>
